lib/datasource/pypi/index.js

```diff
@@ -35,8 +35,14 @@ async function getPkgReleases({ compatibility, lookupName, registryUrls }) {
   if (process.env.PIP_INDEX_URL) {
     hostUrls = [process.env.PIP_INDEX_URL];
   }
-  for (const hostUrl of hostUrls) {
-    const dep = await getDependency(lookupName, hostUrl, compatibility);
+  for (let hostUrl of hostUrls) {
+    hostUrl += hostUrl.endsWith('/') ? '' : '/';
+    let dep;
+    if (hostUrl.endsWith('/simple/')) {
+      dep = getSimpleDependency(depName, hostUrl);
+    } else {
+      dep = await getDependency(depName, hostUrl, compatibility);
```

Comment:
Line 42 and 44 throw on `depName` — i guess this should be `lookupName`, if i get it right?

Reply:
Yes you’re right